src/components/form/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+
+import "./form.scss";
+
+// class Form extends React.Component {
+//   handleSubmit = (e) => {
+//     e.preventDefault();
+//     const formData = {
+//       method: "GET",
+//       url: "https://pokeapi.co/api/v2/pokemon",
+//     };
+//     this.props.handleApiCall(formData);
+//   };
+
+//   render() {
+//     return (
+//       <>
+//         <form onSubmit={this.handleSubmit}>
+//           <label>
+//             <span>URL: </span>
+//             <input name="url" type="text" />
+//             <button type="submit">GO!</button>
+//           </label>
+//           <label className="methods">
+//             <span id="get">GET</span>
+//             <span id="post">POST</span>
+//             <span id="put">PUT</span>
+//             <span id="delete">DELETE</span>
+//           </label>
+//         </form>
+//       </>
+//     );
+//   }
+// }
+
+const Form = ({ handleApiCall }: any) => {
+  const handleSubmit = (e: React.FormEvent<HTMLFormElement>): void => {
+    e.preventDefault();
+    const formData = {
+      method: "GET",
+      url: "https://pokeapi.co/api/v2/pokemon",
+    };
+    return handleApiCall(formData);
+  };
+
+  return (
+    <>
+      <form onSubmit={handleSubmit}>
+        <label>
+          <span>URL: </span>
+          <input name="url" type="text" />
+          <button type="submit">GO!</button>
+        </label>
+        <label>
```

Comment:
Inside the <label> put className="methods", because as of right now the scss file can not find the class label 
 'methods' so it can not apply the style to it.